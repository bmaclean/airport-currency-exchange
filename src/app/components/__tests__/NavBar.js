import React from 'react';
import {shallow} from 'enzyme';
import {NavBar} from '../';

describe('AppHeader', () => {
	describe('renders', () => {
		it('without props', () => {
			const node = shallow(<NavBar />);
			expect(node).toMatchSnapshot();
		});
	});
});
