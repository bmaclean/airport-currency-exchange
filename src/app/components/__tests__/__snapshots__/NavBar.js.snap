// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader renders without props 1`] = `
<NavBar
  classes={
    Object {
      "navBarContainer": "NavBar-navBarContainer-1",
      "tab": "NavBar-tab-2",
      "tabBar": "NavBar-tabBar-3",
    }
  }
/>
`;
